src/routes/api/index.ts

```diff
@@ -1,9 +1,14 @@
 import { Router } from 'express';
+import { createRoom, queryRoom } from '../../controllers/room';
 
 const router = Router();
 
 router.get('/', (req, res) => {
     res.send('Hello world!');
 });
 
+// room module routes
```

Comment:
Let's make a new Router in the room controller and use said router in this file instead of laying out every endpoint in one file.